en/test_and_ci/integration_testing.md

```diff
@@ -6,7 +6,12 @@ This topic explains how to run (and extend) PX4's ROS-based integration tests.
 [MAVSDK Integration Testing](../test_and_ci/integration_testing_mavsdk.md) is preferred when writing new tests.
 Use the ROS-based integration test framework for use cases that _require_ ROS (e.g. object avoidance).
 
-All PX4 integraton tests are executed automatically by our [Continuous Integration](../test_and_ci/continous_integration.md) system.
+All PX4 integration tests are executed automatically by our [Continuous Integration](../test_and_ci/continous_integration.md) system.
+:::
+
+:::note
+This page is about MAVROS integration tests.
```

Comment:
These are actually ROS `1` and MAVROS tests right - not ROS 2 even? What's the story going forward - we'll be migrating them to ROS 2 or dumping them or what?

Reply:
Yes exactly. I'm not sure it's worth keeping them, they have not been worked on in quite some time. MAVROS will still be used for some time I think.